src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -4132,14 +4178,22 @@ private class ConvertCheckingForCustomConverter
         }
 
         #region Delegates converting null
-        private static object ConvertNullToNumeric(object valueToConvert,
-                                                   Type resultType,
-                                                   bool recursion,
-                                                   PSObject originalValueToConvert,
-                                                   IFormatProvider formatProvider,
-                                                   TypeTable backupTable)
+        private static object ConvertNullToNumeric(
+            object valueToConvert,
+            Type resultType,
+            bool recursion,
+            PSObject originalValueToConvert,
+            IFormatProvider formatProvider,
+            TypeTable backupTable)
         {
             typeConversion.WriteLine("Converting null to zero.");
+
+            // Handle BigInteger first, as it is not IConvertible
+            if (resultType == typeof(BigInteger))
+            {
+                return default(BigInteger);
```

Comment:
I guess `default(BigInteger) == BigInteger.Zero`, right? It might be more clear to use `BigInteger.Zero` here.

Reply:
```suggestion
                return BigInteger.Zero;
```